### DISCUSSION TITLE
Package roadmap

### DISCUSSION THREAD

**Discussion by austincarpenter** [rawilk/profile-filament-plugin#22] [General]
Hey there, 
Awesome, feature-rich package. Would really like to incorporate into a production application once a stable release is published. What's the roadmap at this stage?
Austin

**Comment by rawilk** [comment]
I'm not 100% sure, but I do hope to have a stable release ready by Q3 or Q4 of this year. IMO it's pretty close to being ready for a stable release, however I do want to revisit my implementation of my sudo mode action before that point, now that I've gotten a better understanding of how Filament works.

**Comment by rawilk** [comment]
I haven't been able to give this package the attention it needs until now, but it is much closer to a stable release than it's ever been. With the refactoring I've done in v0.7.0, I don't plan on making any more major breaking changes to the API before I tag a stable release. 

I plan on testing this out on some non-critical production sites and will hopefully be ready for a stable release within the next couple months.
